<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <item>
       <widget class="QSplitter" name="left_right">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QTextBrowser" name="cn_input">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextBrowser" name="vn_output">
            <property name="openLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,2">
            <item>
             <widget class="QPushButton" name="previous_page">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Previous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="current_page">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="next_page">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottom_bar">
        <item>
         <widget class="QProgressBar" name="progress_bar">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="char_per_page">
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>50000</number>
          </property>
          <property name="singleStep">
           <number>1000</number>
          </property>
          <property name="value">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="read_from_clipboard"/>
    <addaction name="read_from_file"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="save_to_file"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="read_from_clipboard">
   <property name="text">
    <string>From clipboard</string>
   </property>
  </action>
  <action name="read_from_file">
   <property name="text">
    <string>From file</string>
   </property>
  </action>
  <action name="save_to_file">
   <property name="text">
    <string>To file</string>
   </property>
  </action>
  <action name="save_to_clipboard">
   <property name="text">
    <string>To clipboard</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
